text/ProofModes.md

```diff
@@ -0,0 +1,118 @@
+# Proof Modes
+
+This CEP is about the infrastructure needed in order to have multiple 
+proof modes coexisting in a modular way (i.e. without syntactic or semantic conflicts)
+
+## Status quo (as per Coq 8.6)
+
+1. Tactic names are kernel names, i.e. unique, and can be accessed this way
+
+    ```
+    foo.tac arg1.
+    bar.tac arg1.
+    ```
+    
+1. Linking (not even requiring) a plugin loads into the *lexer* the tokens 
+   specific to the grammar implemented by the plugin
+   
+1. Linking (not even requiring) a plugin loads into the *parser* grammar the rules
+   specific to the plugin 
+   
```

Comment:
I just wanted to point out that as far as I understand the thing, it is not like it was not supported, but that the current official way is that you are supposed to do galatic hacks everytime you care about state in Coq.